content/hi/docs/contribute/review/for-approvers.md

```diff
@@ -0,0 +1,202 @@
+---
+title: अनुमोदकों और समीक्षकों के लिए समीक्षा करना
+linktitle: अनुमोदकों और समीक्षकों के लिए
+slug: के लिए अनुमोदनकर्ता
+content_type: संकल्पना
+weight: 20
+---
+
+<!-- अवलोकन -->
+
+SIG Docs [समीक्षक](/docs/contribute/participate/#reviewers) और
+[अनुमोदक](/docs/contribute/participate/#approvers) कुछ अतिरिक्त चीज़ें करें
+परिवर्तन की समीक्षा करते समय।
+
+प्रत्येक सप्ताह एक विशिष्ट दस्तावेज़ अनुमोदक स्वयंसेवकों को ट्राइएज करने के लिए और पुल अनुरोधों की समीक्षा करें। यह व्यक्ति सप्ताह के लिए "पीआर रैंगलर" है। देखें [पीआर रैंगलर अनुसूचक](https://github.com/kubernetes/website/wiki/PR-Wranglers) अधिक जानकारी के लिए। पीआर रैंगलर बनने के लिए, साप्ताहिक एसआईजी डॉक्स मीटिंग में भाग लें और स्वयंसेवक बनें। यहां तक कि अगर आप वर्तमान सप्ताह के लिए शेड्यूल पर नहीं हैं, तब भी आप पुल की समीक्षा कर सकते हैं अनुरोध (पीआर) जो पहले से सक्रिय समीक्षा के अधीन नहीं हैं।
+
+
+रोटेशन के अलावा, बॉट समीक्षकों और अनुमोदकों को असाइन करता है प्रभावित फाइलों के मालिकों के आधार पर पीआर के लिए।
+
+<!-- body -->
+
+## एक पीआर की समीक्षा करना
+
+कुबेरनेट्स प्रलेखन इस प्रकार है [कुबेरनेट्स कोड समीक्षा प्रक्रिया](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process).
+
+में सब कुछ वर्णित है [एक पुल अनुरोध की समीक्षा करना](/docs/contribute/review/reviewing-prs) लागू होता है, लेकिन समीक्षकों और अनुमोदकों को निम्नलिखित कार्य भी करने चाहिए:
+
+- का उपयोग `/assign` जरूरत पड़ने पर पीआर को एक विशिष्ट समीक्षक सौंपने के लिए प्रो कमांड। यह अतिरिक्त महत्वपूर्ण है जब कोड योगदानकर्ताओं से तकनीकी समीक्षा का अनुरोध करने की बात आती है।
+
+  {{< note >}}
+  की ओर देखने के लिए `reviewers` यह देखने के लिए कि तकनीकी समीक्षा कौन प्रदान कर सकता है, मार्कडाउन फ़ाइल के शीर्ष पर फ्रंट-मैटर में फ़ील्ड।
+  {{< /note >}}
+
+- यह सुनिश्चित करना कि पीआर अनुसरण करता है [विषय](/docs/contribute/style/content-guide/) और [शैली](/docs/contribute/style/style-guide/) गाइड; यदि ऐसा नहीं होता है तो लेखक को गाइड के प्रासंगिक भाग से लिंक करें।
+- पीआर लेखक को परिवर्तनों का सुझाव देने के लिए लागू होने पर GitHub **परिवर्तन का अनुरोध करें** विकल्प का उपयोग करना।
```

Comment:
Since it is a GitHub option/feature, it is better to keep the `Request Changes` as it is.
```suggestion
- यदि लागू हो, तो GitHub **Request Changes** विकल्प का उपयोग करके PR लेखक को परिवर्तनों का सुझाव दें।
```